bundles/bundle.properties

```diff
@@ -109,12 +109,12 @@ block.heavy-armaments-t1_duo.details = An Upgrade to the popular 'Duo'. Shoots l
 
 ;Missiles
 block.heavy-armaments-t1_missile.name = Hellseeker
-block.heavy-armaments-t1_missile.description = Simple missile launcher, has a huge range.
+block.heavy-armaments-t1_missile.description = A simple missile launcher which has a lot of range.
 block.heavy-armaments-t1_missile.details = Standard issue surface to air missile launcher. Old and reliable. 
 
 block.heavy-armaments-t2_missile.name = Stiletto
-block.heavy-armaments-t2_missile.description = Fires missiles in short burst
-block.heavy-armaments-t2_missile.details = Many micro missiles were bound together before firing, enabling multiple missiles to be fired at quick succession, although due to this design it loses velocity rather quickly. n/Turret by sh1penfire
+block.heavy-armaments-t2_missile.description = Fires missiles in short bursts.
+block.heavy-armaments-t2_missile.details = Many micro missiles were bound together before firing, allowing it multiple missiles to be fired at quick succession, although due to this design it loses velocity rather quickly. n/Turret by sh1penfire.
```

Comment:
This part sounds weird, are you sure?
Thank you for unbutchering my terrible english btw

Reply:
got confused redoing that part